migrations/20190410101237-fix-localField-type.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    async up (db) {
+        var operations = [];
+
+        const ObjectID = require('mongodb').ObjectID;
+        const reports = await db.collection('wst_Reports').find().toArray();
+        reports.forEach(report => {
+            var layerID = report.selectedLayerID;
+            operations.push({
+                'updateOne': {
+                    'filter': {'_id': report._id},
+                    'update': {
+                        '$set': {'selectedLayerID': ObjectID(layerID)}
+                    }
+                }
+
+            });
+        });
+
+        db.collection('wst_Reports').bulkWrite(operations);
+    },
+
+    down (db) {
+    }
```

Comment:
It is possible to rollback the migration here, so you must also write the "down" part